docs/security.md

```diff
@@ -838,6 +838,18 @@ The following options provides finer-grained control for this feature:
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.yarn.kerberos.renewal.excludeHadoopFileSystems</code></td>
```

Comment:
this should be in the yarn specific kerberos section -> YARN-specific Kerberos Configuration (http://spark.apache.org/docs/latest/running-on-yarn.html). and we should link to it from the security.md doc i

Reply:
Moved. Thanks.